CHANGELOG.md

```diff
@@ -0,0 +1,319 @@
+# Changelog
```

Comment:
Is there a way to bring the `master` branch here up to date by merging upstream rather than basically squashing all of the changes since the fork was made into a single commit?

Reply:
Ah, I see that Mark's Python 3.8 changes were merged to `master` in this fork, which makes that more complicated.  Ideally, we could reset `master` to point to the same commit as upstream and work from there.